PokeOneWeb/Frontend/pokeoneweb/src/app/core/components/placed-item-list/placed-item-list.component.ts

```diff
@@ -19,11 +19,27 @@ export class PlacedItemListComponent implements OnInit {
 
     ngOnInit(): void {
         this.model.placedItems = this.placedItems;
+        this.checkItemNames();
         this.checkLocations();
         this.applyInitialSorting();
     }
 
-    private checkLocations() {
+    public checkItemNames() {
+        if (!this.model.placedItems) return;
+
+        const itemNameList = [];
+        for (const item of this.model.placedItems) {
+            itemNameList.push(item.itemName);
+        }
```

Comment:
mit einem stream lösen... also this.model.placedItems.map((placedItem) => placedItem.itemName)